docs/graphics/HomePageIcons.js

```diff
@@ -0,0 +1,414 @@
+// @flow strict
+import { type Node } from 'react';
+
+export function HowToWork(): Node {
+  return (
+    <svg
```

Comment:
@ayeshakmaz since these are no longer just icons, could we import svg files instead? E.g. by using `babel-plugin-react-svg`

That way, it'll also be easier to see a before / after when we update the SVG

Reply:
This is a new package correct? We don't currently use it anywhere?